protocol/0056-REWA-rewards_overview.md

```diff
@@ -1,73 +1,120 @@
+-  `quantum` is an asset level field described in the [asset framework](0040-ASSF-asset_framework.md)
 
-Trading reward accounts are defined by the payment asset (the asset in which the reward is paid out), the market in scope, and the reward type (metric). I.e. there can be multiple rewards with the same type paid in different assets for the same market.
+The reward metric for the single *market creator* party is as follows:
 
-It must be possible for any party to run a one off [transfer](0057-TRAN-transfers.md) or create a [recurring transfer](0057-TRAN-transfers.md) to any of these reward accounts. 
-Note that the market settlement asset has nothing to do in particular with the asset used to pay out a reward for the market for any of the relevant trading rewards. 
+- **IF** `cumulative volume < rewards.marketCreationQuantumMultiple * quantum` **THEN** `market creation metric := 0`
+- **ELSE** `market creation metric := 1` (NB: this is 1 as market creation rewards are paid equally to all qualifying creators for *reaching* the volume threshold, not pro-rata based on cumulative volume)
+
+When the `market creation metric` for a party is `>0` and the reward account balance for a specific reward asset is also `>0` (i.e. when a creator is rewarded):
+
+- A flag is added for each of the recorded `funders` of the reward account paired with the market and reward asset.
+This flag is used to prevent a recurring transfer from funding a creation reward in the same asset more than once.
+See the [transfers](./0057-TRAN-transfers.md) spec.
+- The list of funders for the reward account is cleared after the account is emptied (i.e. after one or more parties are rewarded for market creation).
+
+Market creation reward metrics (both each market's `cumulative volume` and the `payout record flags` to identify [market, payout asset, funder] combinations that have already been rewarded) are stored in [LNL checkpoints](./0073-LIMN-limited_network_life.md) and will be restored after a checkpoint restart.
+
+
+
+## Reward accounts
+
+Trading reward accounts are defined by the reward asset (the asset in which the reward is paid out), the market, and the reward type (metric). 
+That is, there can be multiple rewards with the same type paid in different assets for the same market.
+
+Note that the market settlement asset has nothing to do in particular with the asset used to pay out a reward for a market. 
+That is, a participant might recieve rewards in the settlement asset of the market, in VEGA governance tokens, and in any number of other unrelated tokens (perhaps governance of "loyalty"/reward tokens issued by LPs or market creators, or stablecoins like DAI).
+
+Reward accounts can be individually funded by normal single transfers. They can also be funded on an ongoing (recurring) basis in groups, pro-rata by their relative total reward metrics for the given reward type. See [transfers](./0057-TRAN-transfers.md).
```

Comment:
one off currently do not support metrics. the same can be achieved though by a recurring transfer with an end epoch being the following one